src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,6 +1,36 @@
 import React from 'react';
-import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+import 'bulma/css/bulma.min.css';
+import { Box, Block, Heading } from 'react-bulma-components';
+
+import { CommentList } from '../CommentList/CommentList';
+import { Comment } from '../CommentInfo/CommentInfo';
+import { UserInfo, User } from '../UserInfo/UserInfo';
+
+export interface Post {
+  userId: number,
+  id: number,
+  title: string,
+  body: string,
+  user: User | undefined,
```

Comment:
```suggestion
  user: User | null,
```
that's a common practice to do this way